rust/windows-client/README.md

```diff
@@ -2,14 +2,28 @@
 
 This crate houses a Windows GUI client.
 
-## Building
+## Setup
 
-From this dir:
+1. Install the latest
+   [Visual Studio Community Edition](https://visualstudio.microsoft.com/downloads/)
+   and ensure **C++ support with Clang** is selected. Note: Visual Studio Code
+   won't have clang -- you need actual Visual Studio.
+1. Install rustup.
```

Comment:
I still kinda think Rustup installs VS on behalf of the user, I don't see a VS installer in my Downloads folder, but I also don't have a fresh VM to test it on

Reply:
https://rust-lang.github.io/rustup/installation/windows-msvc.html#automatic-install
We could keep these steps but just put rustup first and make VS "If Rustup didn't handle this"
I think rustup installs the correct toolchain / target by default too

Reply:
I can't figure out how to propose code changes in the Github UI

Reply:
btw I might have an x64 Windows Server 2022 VM going soon that I can use to test prod builds / fresh developer setups

Reply:
> I still kinda think Rustup installs VS on behalf of the user

Maybe it installs the VS build tools? I would have thought installing VS would prompt for some kind of end-user license agreement.

In any case I removed this step for now since I was only using it to install clang.